### PR TITLE
Changed the data copy mechanism to work with chum/cli.

### PR DESCRIPTION
**Pull request by poetaster**
If you wish, I can take over maintenance? I have this building on obs: https://build.sailfishos.org/package/show/home:poetaster:coderus/MachinesVsMachines.

### CODE REVIEW

**Comment by poetaster** [comment]
With these changes, the working game is available on Chum. I can maintain it there if you wish.

File: rpm/machines-vs-machines-sfos.spec [hunk 1]
```diff
@@ -14,22 +14,45 @@ Name:       harbour-machines-vs-machines-sfos
...
+PackageName: Machines vs. Machines
+Type: desktop-application
+Categories:
+ - Game
+PackagerName: Mark Washeim (poetaster)
+DeveloperName: Michael Zanetti
+Custom:
+ - Repo: https://github.com/mzanetti/machines-vs-machines-sfos
+ - PackagingRepo: https://github.com/poetaster/machines-vs-machines-sfos
+Icon: https://github.com/poetaster/machines-vs-machines-sfos/raw/master/icons/256x256/harbour-machines-vs-machines-sfos.png
```
Comment (mzanetti) [inline rpm/machines-vs-machines-sfos.spec]: Those should probably point to this repository instead, no?

File: rpm/machines-vs-machines-sfos.spec [hunk 2]
```diff
@@ -14,22 +14,45 @@ Name:       harbour-machines-vs-machines-sfos
...
+Screenshots:
+ - https://raw.githubusercontent.com/poetaster/machines-vs-machines-sfos/master/screen-1.png
+ - https://raw.githubusercontent.com/poetaster/machines-vs-machines-sfos/master/screen-2.png
+ - https://raw.githubusercontent.com/poetaster/machines-vs-machines-sfos/master/screen-3.png
+ - https://raw.githubusercontent.com/poetaster/machines-vs-machines-sfos/master/screen-4.png
+Url:
+ - Help: https://github.com/poetaster/machines-vs-machines-sfos/discussions
+ - Bugtracker: https://github.com/poetaster/machines-vs-machines-sfos/issues
+ - Donation: https://www.paypal.me/poetasterFOSS
```
Comment (mzanetti) [inline rpm/machines-vs-machines-sfos.spec]: Same, I suppose... And for paypal... I find it a bit bold to silently take over the donation url :)

**Review by mzanetti — COMMENTED** [review]
Hey, nice! 

The changes in the license are contradicting. the .spec file says GPL3 while the Licence file is set to BSD. GPL3 it should be.

File: rpm/machines-vs-machines-sfos.spec (hunk 2 above)
Comment (poetaster) [reply rpm/machines-vs-machines-sfos.spec]: Yeah, that was NOT intentional. Template. I'm going to remove that. Now.

Comment (poetaster) [reply rpm/machines-vs-machines-sfos.spec]: I'm not sure that your original paypal link works any more? Let me know what it should be.

**Comment by poetaster** [comment]
Ok, as usual, I got a bit carried away. So, I had fixed the visibility of bullet's issue and downsampled some audio (with high fire rates had drop outs). Then I got to png optimizing. I got the install size down from 36mb to 22mb. But I'm not sure any of this interests you, or worse, may be at odds with your intentions?

File: rpm/machines-vs-machines-sfos.spec (hunk 2 above)
Comment (poetaster) [reply rpm/machines-vs-machines-sfos.spec]: Ah. Just tested https://www.paypal.com/donate/?cmd=_s-xclick&hosted_button_id=USWAUUU2MD9Z4 ... so I'll put that in place.

**Comment by poetaster** [comment]
The other thing is, I would guess that doing SFOS specific support ist not something you want to do? Or?

File: rpm/machines-vs-machines-sfos.spec (hunk 1 above)
Comment (poetaster) [reply rpm/machines-vs-machines-sfos.spec]: I can make all the links point here. Do you want me to skip discussions? (not enabled here).

Comment (poetaster) [reply rpm/machines-vs-machines-sfos.spec]: I now have all (except the screen snaps) pointing to you. Shall I update the screens, too? The current form is a 'hodge ;podge' but it does allow separation of concerns :) https://github.com/poetaster/machines-vs-machines-sfos/blob/e742839144ed451e695e0c8534045211e08c5d9f/rpm/machines-vs-machines-sfos.spec 

I'm in discussion with people about how to manage level packages to reduce the size. Sounds antiquated but is still an issue on sfos.

**Comment by poetaster** [comment]
The packages are in Chum at: https://build.sailfishos.org/package/view_file/sailfishos:chum:testing/MachinesVsMachines./_service:tar_git:machines-vs-machines-sfos.spec?expand=1

I've done a release/build just to get rid of the donation stuff, although the link is not currently shown in the Chum gui. Now, it should be. The issue tracker will point here now, but discussions still point to my fork. Hope I'm not overcomplicating things.

And thank you!

**Comment by mzanetti** [comment]
Well, tbh, I moved away from Ubuntu Phone and SFOS so I'm totally fine, actually happy, that someone will look after this. Also I don't want to overcomplicate things, so whatever works for you in getting this out to the users, fine with me.

Not that it would give lots of donations (actually pretty close to zero) but if by any chance that would change because of you taking care of it, I'd gladly share.

For publishing it to the store... I guess since I uploaded the initial build I'd have to do that in order for it to be an update and not a new app, but It's been a while since I did that so I would need to read up on stuff again. What would be the next steps then? Anything I should do? If you're on discord or telegram we can also chat if that makes it easier than github comments (check out nymea and its communication channels, you can find me there).

**Comment by poetaster** [comment]
I figured you'd probably not have devices ;) I was an early adopter who was away a couple of years. But since the volla phone and a number of community ports have  become viable, I've taken over a bunch of projects. 

In the main, I'm happy just maintaining it as is. I wanted to see if any other users are interested in building levels. So maybe something happens on that front. 

I don't need any money :) I think you had a bunch of collaborators you can buy a beer for if anything should ever come in :)

If you still have you're old harbour account, it's really easy. I need to fix one little glitch (one wrongly sized icon). Then I'll do a release with the fix and add the rpms to the release. You could then download those and just upload em. Sadly, you can't pull the ones from jolla's obs because, ironically, that's not supported :)

**Pull request merged**